docs/intro/tutorial01.txt

```diff
@@ -79,14 +85,14 @@ Let's look at what :djadmin:`startproject` created:
 
 These files are:
 
-* The outer :file:`mysite/` root directory is a container for your project. Its
+* :file:`djangotutorial/`: A root directory that contains your project. Its
   name doesn't matter to Django; you can rename it to anything you like.
```

Comment:
Is this bullet needed and is ```Let's look at what :djadmin:`startproject` created:` showing `djangotutorial/``` still valid?
The command no longer created the directory so we might want to adjust this. Maybe moving the "name doesn't matter to Django" further up to when we created the directory

Reply:
Moved it up to wen we created the drectory.

Reply:
Thank you @petmetic! Did you commit/push this change? I'm not seeing it :thinking:

Reply:
Pushed it now. ad a bit of trouble and had to rebase it... sorry. 
Hope everything is ok now. Please let me know if I there is anything else.

Reply:
@sarahboyce I have re-reviewed the latest version and I think your comment is spot on. Since `djangotutorial` was created by the person following the tutorial, is not "created by `startproject`". I have tweaked and refactored things a bit. Thanks!